meson.build

```diff
@@ -115,7 +115,7 @@ cdata.set('EXV_HAVE_ICONV', iconv_dep.found())
 cdata.set('EXV_HAVE_LIBZ', zlib_dep.found())
 cdata.set('EXV_USE_CURL', curl_dep.found())
```

Comment:
In addition, might want to gate EXV_USE_CURL by the webready option as well, as it's done in the CMake configuration?

Reply:
Yes I thought about that too for consistency with the actual `EXV_USE_CURL`. Done on the force push below.